pkg/apis/v1alpha1/kwokctl_configuration_types.go

```diff
@@ -143,7 +143,7 @@ type KwokctlConfigurationOptions struct {
 	KubeSchedulerImage string `json:"kubeSchedulerImage,omitempty"`
 
 	// KwokControllerImage is the image of Kwok.
-	// is the default value for flag --contoller-image and env KWOK_CONTROLLER_IMAGE
+	// is the default value for flag --controller-image and env KWOK_CONTROLLER_IMAGE
```

Comment:
I think these 2 typo fixes can be submitted separately.

Reply:
https://github.com/kubernetes-sigs/kwok/pull/317